### ISSUE TITLE
conn.once('end', function() {

### ISSUE THREAD

**Issue by FLYBYME** [feross/bittorrent-swarm#20]
Found this error.

```
C:\node\BT-Tracker\node_modules\bittorrent-swarm\lib\peer.js:93
        conn.once('end', function() {
            ^
TypeError: Cannot read property 'once' of null
    at Peer.onConnect (C:\node\BT-Tracker\node_modules\bittorrent-sw
    at Socket.<anonymous> (C:\node\BT-Tracker\node_modules\bittorren
    at Socket.g (events.js:199:16)
    at Socket.emit (events.js:104:17)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1001:10)
```

**Comment by feross** [comment]
Will look into this.

**Comment by feross** [comment]
Fixed in 5.0.4.
